src/gettingstarted/cli/api-tokens.md

```diff
@@ -0,0 +1,63 @@
+# API Tokens
+
+## Obtaining a token
+
+The Platform.sh CLI can also be used from CI services or other automation tools, and supports an API Token authentication option for that purpose.
+
+An API token can be created through the UI.  Go to the "User" page from your accounts drop-down, then select the "Account Settings" tab, then the "API Tokens".
+
+Click the "Create an API Token" link.  You may be asked to reverify your password, then enter a unique application name to identify the token.
+
+![API Token screen](/images/api-token-list.png)
+
+After creating the token it will be displayed once at the top of the page in a green banner.  You may also view it later by clicking the "view" link next to the token name.  You will be asked to reverify your password as well when viewing the token.
+
+Now set that token to an environment variable named `PLATFORMSH_CLI_TOKEN` on your system where the CLI will run.  Consult the documentation for your CI system to see how to do that.
+
+> **note**
+>
+> If running CLI commands from any automated system, including a Platform.sh cron task, we urge you to use the `--no-wait` flag on any commands that may take more than a second or two to avoid blocking the process.
+
+## Automation users
+
+For projects with more than one user it is often a good idea to create a dedicated "automation user" to run automation tasks such as backups.  That way, that user can be given a very restrictive set of permissions limited to just its needed tasks.  It will also show up in logs and activity streams as a separate entry from human users.
+
+Every user account needs its own unique email address.  This account will not need an SSH key, however.  The permissions it needs on a given project will depend on what automation tasks it will be running.  Backups, for instance, require `admin` access, while checking out code from a CI server to run tests on it would require only `reader` access.
```

Comment:
"will not need" -> "may not need" (it would need a key for `git clone`, for example, or `platform ssh ...`)